pnpm-workspace.yaml

```diff
@@ -6,7 +6,7 @@ packages:
 catalog:
   '@cloudflare/workers-types': ^4.20241112.0
   'typescript': ^5.6.2
-  'vite': '6.0.0-beta.10'
+  'vite': '/Users/dario/Repos/my-repos/vite/packages/vite'
```

Comment:
Clearly this PR won't be ready to merge until the version of vite with the `resolve.builtins` config is (hopefully) released